libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -213,17 +226,26 @@ class AP_Airspeed
         Airspeed_Calibration calibration;
         float last_saved_ratio;
         uint8_t counter;
+        uint8_t reenable_use;
```

Comment:
reenable_use should signed to match AP_Int8 use

Reply:
Sure no probs